utils/kvrocks2redis/sync.cc

```diff
@@ -104,98 +78,68 @@ void Sync::Stop() {
   LOG(INFO) << "[kvrocks2redis] Stopped";
 }
 
-Status Sync::auth() {
-  // Send auth when needed
-  if (!config_->kvrocks_auth.empty()) {
-    const auto auth_command = redis::MultiBulkString({"AUTH", config_->kvrocks_auth});
-    auto s = util::SockSend(sock_fd_, auth_command);
-    if (!s) return s.Prefixed("send auth command err");
-    std::string line = GET_OR_RET(util::SockReadLine(sock_fd_).Prefixed("read auth response err"));
-    if (line.compare(0, 3, "+OK") != 0) {
-      return {Status::NotOK, "auth got invalid response"};
-    }
-  }
-  LOG(INFO) << "[kvrocks2redis] Auth succ, continue...";
-  return Status::OK();
+Status Sync::tryCatchUpWithPrimary() {
+  auto s = storage_->GetDB()->TryCatchUpWithPrimary();
+  return s.ok() ? Status() : Status::NotOK;
```

Comment:
```suggestion
  return storage_->GetDB()->TryCatchUpWithPrimary();
```

Why not just return it?

Reply:
Ahh I see. It's rocksdb::Status.